documentation/src/components/page.ts

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { html, TemplateResult, query } from 'lit-element';
+import { LayoutElement } from './layout';
+
+export class Page extends LayoutElement {
+    @query('#layout-content')
+    contentElement?: HTMLDivElement;
+
+    public renderContent(): TemplateResult {
+        return html`
+            <slot></slot>
+        `;
+    }
+
+    protected firstUpdated(): void {
+        const link = document.createElement('link');
+        link.rel = 'stylesheet';
+        link.href = 'https://use.typekit.net/evk7lzt.css';
```

Comment:
I notice that there is still another typekit link above. Do we still need [this one](https://github.com/adobe/spectrum-web-components/blob/e93c5b17491364105cb7d2999439341e6400c948/documentation/index.html#L68-L72)

Reply:
I'm not 100% up-to-date on using Typekit, but it does seem that if you remove the JS the CSS still works. Done.